frontend/src/Interface/InputModuleProps.tsx

```diff
@@ -7,10 +7,12 @@
  * @author    @GDamaso
  */
 import FarmDetailsInterface from './FarmDetailsInterface';
+import { TempNutrientsInterface } from './NutrientsInterface';
 
 interface InputModuleProps {
   farmDetails: FarmDetailsInterface;
   updateFarmDetails: (farmDetails: FarmDetailsInterface) => void;
+  updateNutrientDetails: (nutrientDetails: TempNutrientsInterface) => void;
```

Comment:
This should be optional, only fertilizers and calculation need it

Reply:
My answer is from the previous comment.